src/api/v3/locations.py

```diff
@@ -0,0 +1,68 @@
+from django.http import JsonResponse
+
+from cms.models import POI
+from ..decorators import json_response
+
+
+def transform_poi_translation(poi_translation):
+    """
+    Function to create a JSON from a single poi_translation object.
+
+    :param poi_translation: The poi translation object which should be converted
+    :type poi_translation: ~cms.models.pois.poi_translation.POITranslation
+
+    :return: return data necessary for API
+    :rtype: dict
+    """
+
+    # is there a better way to access this?
+    try:
+        icon_url = poi_translation.icon.url
+    except AttributeError:
+        icon_url = ""
+
+    poi = poi_translation.poi
+    return {
+        "id": poi_translation.id,
+        "url": poi_translation.permalink,
+        "path": poi_translation.slug,
+        "title": poi_translation.title,
+        "modified_gmt": poi_translation.last_updated,
+        "excerpt": poi_translation.short_description,
+        "content": poi_translation.description,
+        "available_languages": poi_translation.available_languages,
+        "thumbnail": icon_url,
+        "location": {
+            "id": poi.id,
+            "name": poi_translation.title,
+            "address": poi.address,
+            "town": poi.city,
+            "state": None,
+            "postcode": poi.postcode,
+            "region": poi.region.slug,
```

Comment:
```suggestion
            "region": None,
```
I think the field `region` of the WordPress events plugin means a different thing than what we refer to as "region"... so I would just set this field to `None` as well... (I couldn't find a location in wordpress which had set any value in this field)